app-frontend/src/app/components/lab/nodeCreateSidebar/nodeCreateSidebar.js

```diff
@@ -108,19 +97,23 @@ class NodeCreateSidebarController {
         this.opChars = allowedOps.map((ao) => ao.op);
     }
 
-    fetchProjectPage(page = 1) {
+    fetchProjectPage(searchVal, page = 1) {
         if (this.loadingList) {
             return;
         }
+        const params = {
```

Comment:
this exact code exists in two places - might as well make a `createSearchParams` function